FirstProject/BaseAlgorithms.cs

```diff
@@ -31,17 +32,28 @@ public int GetPower(int value1, int value2)
             }
             return power;
         }
-       public bool IsPerfectNumber(int value)
+        public bool IsPerfectNumber(int value)
         {
             int sum = 0;
             for (int i = 1; i < value; i++)
             {
-                if (value % i == 0)  
+                if (value % i == 0)
                     sum += i;
             }
             return sum == value;
-                return true;
+            return true;
+        }
+        public int Average(int[] value)
+        {
+            int sum = 0;
+            for (int i = 0; i < value.Length; i++)
+            {
+                sum += value[i];
+            }
+            int average = sum / value.Length;
```

Comment:
A nie lepiej zwrócić tam po prostu wartość sum / length zamiast przypisywać to do zmiennej